src/app/dspace/components/item/collection.component.ts

```diff
@@ -1,33 +1,28 @@
-import {Component, Input} from 'angular2/core';
+import {Component} from 'angular2/core';
+import {ROUTER_DIRECTIVES} from 'angular2/router';
 import {TranslatePipe} from "ng2-translate/ng2-translate";
 
-import {Collection} from "../../models/collection.model"
+import {Collection} from "../../models/collection.model";
 import {ViewElementComponent} from './view-element.component';
 
 /**
  * Component for the collections of the simple-item-view.
- * When you click on the collection name, it has to redirect to the right collection.
  */
 @Component({
     selector: 'item-collection',
-    inputs: ['itemData'],
-    directives: [ViewElementComponent],
+    inputs: ['itemParent'],
+    directives: [ROUTER_DIRECTIVES, ViewElementComponent],
     pipes: [TranslatePipe],
-    template:
-        `
-        <view-element [header]="componentTitle| translate">
-             <a [attr.href]="collectionURIPrefix+itemData.id">{{ itemData.name }}</a>
-        </view-element>
-        `
+    template: `
+                <view-element [header]="componentTitle| translate">
+                    <a *ngIf="itemParent && itemParent.component && itemParent.id" [routerLink]="[itemParent.component, {id: itemParent.id}]">{{ itemParent.name }}</a>
```

Comment:
Good point. I will do another pass to do this. I also wanted to add all the typed returns of functions. I noticed you did this in several classes and is better TypeScript coding. Lets the transpiler catch more errors.